<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>846</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <attribute name="title">
        <string>Search</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLineEdit" name="lineEdit">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>45</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>Enter hashtag here</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Search">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>Search</string>
            </property>
            <property name="shortcut">
             <string>Return</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Exit">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>Exit</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="source">
            <property name="title">
             <string>News Source</string>
            </property>
            <widget class="QRadioButton" name="radioButton">
             <property name="geometry">
              <rect>
               <x>80</x>
               <y>80</y>
               <width>151</width>
               <height>31</height>
              </rect>
             </property>
             <property name="text">
              <string>Indian Express</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
            <widget class="QRadioButton" name="radioButton_2">
             <property name="geometry">
              <rect>
               <x>80</x>
               <y>130</y>
               <width>161</width>
               <height>31</height>
              </rect>
             </property>
             <property name="text">
              <string>The Hindu</string>
             </property>
            </widget>
            <widget class="QRadioButton" name="radioButton_3">
             <property name="geometry">
              <rect>
               <x>80</x>
               <y>240</y>
               <width>131</width>
               <height>31</height>
              </rect>
             </property>
             <property name="text">
              <string>CNN</string>
             </property>
            </widget>
            <widget class="QRadioButton" name="radioButton_4">
             <property name="geometry">
              <rect>
               <x>80</x>
               <y>280</y>
               <width>131</width>
               <height>41</height>
              </rect>
             </property>
             <property name="text">
              <string>abc News</string>
             </property>
            </widget>
            <widget class="QRadioButton" name="radioButton_5">
             <property name="geometry">
              <rect>
               <x>80</x>
               <y>390</y>
               <width>151</width>
               <height>41</height>
              </rect>
             </property>
             <property name="text">
              <string>Google News</string>
             </property>
            </widget>
            <widget class="QLabel" name="label">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>30</y>
               <width>221</width>
               <height>51</height>
              </rect>
             </property>
             <property name="text">
              <string>Local (Indian Source) :</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_2">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>180</y>
               <width>241</width>
               <height>51</height>
              </rect>
             </property>
             <property name="text">
              <string>International (Global Source) : </string>
             </property>
            </widget>
            <widget class="QLabel" name="label_3">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>330</y>
               <width>221</width>
               <height>61</height>
              </rect>
             </property>
             <property name="text">
              <string>ALL (Multi Source) : </string>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Page</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="Refresh">
              <property name="text">
               <string>Refresh</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="Scroll">
              <property name="text">
               <string>Scroll</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="Zoom">
              <property name="text">
               <string>Zoom</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="Cancel">
              <property name="text">
               <string>Cancel</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QWebEngineView" name="view" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="tabletTracking">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>460</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header>qwebengineview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
